src/clients/node/scripts/test_install_on_mac.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
I feel like `/bin/zsh` would be more correct here, cause that's mac, but I didn't do that because shellcheck can't zsh